uber/config.py

```diff
@@ -363,6 +363,8 @@ def _is_intstr(s):
 
 c.TABLE_PRICES = defaultdict(lambda: _config['table_prices']['default_price'],
                              {int(k): v for k, v in _config['table_prices'].items() if k != 'default_price'})
```

Comment:
Ahh, that makes sense, since it also explains why you changed from ``int_options`` to ``options``.

Story checks out!